src/Utilities/ExcelImporterUtils.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace ExcelMapper.Utilities
+{
+    public static class ExcelImporterUtils
+    {
+#if NETSTANDARD2_0
+        public static List<ExcelClassMap> RegisterMapperClassesByNamespace(this ExcelImporter importer,
+            string namespacestr)
+        {
+            var classes = Assembly.GetExecutingAssembly()
+                .GetTypes().AsParallel()
+                .Where(t => t.IsClass && t.Namespace == namespacestr);
+
+            var objects = classes.AsParallel().Select(Activator.CreateInstance)
+                .AsParallel().OfType<ExcelClassMap>().ToList();
```

Comment:
You can remove the `ToList` call as this method should return `IEnumerable` instead of `List`

Reply:
The ToList() prevents it from enumerating multiple times

Reply:
True, this makes sense. Still, let's only return `IEnumerable<ExcelClassMap>` as we don't want to leak the implementation